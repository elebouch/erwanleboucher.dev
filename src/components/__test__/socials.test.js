import React from "react"
import { render } from "@testing-library/react"
import "jest-styled-components"

import Socials from "../socials"

describe("Socials", () => {
  it("renders correctly", () => {
    const { container } = render(<Socials />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
