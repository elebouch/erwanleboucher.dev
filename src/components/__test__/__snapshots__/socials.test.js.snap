// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Socials renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 10px;
}

.c0 > a:not(:first-child) {
  margin-left: 20px;
}

.c1:hover {
  -webkit-filter: grayscale(1);
  filter: grayscale(1);
}

.c1 > svg {
  color: var(--dark);
  height: px;
  -webkit-transition: opacity 0.2s linear;
  transition: opacity 0.2s linear;
  cursor: pointer;
}

.c1 > svg:hover {
  opacity: 0.7;
}

<div
  className="c0"
>
  <a
    aria-label="Github"
    className="c1"
    href="https://github.com/eleboucher"
    rel="noopener noreferrer"
    target="_blank"
  >
    <test-file-stub
      alt="GitHub"
      loading="lazy"
    />
  </a>
  <a
    aria-label="LinkedIn"
    className="c1"
    href="https://www.linkedin.com/in/erwan-leboucher/"
    rel="noopener noreferrer"
    target="_blank"
  >
    <test-file-stub
      alt="LinkedIn"
      loading="lazy"
    />
  </a>
  <a
    aria-label="Mail"
    className="c1"
    href="mailto:erwanleboucher@gmail.com"
    rel="noopener noreferrer"
    target="_blank"
  >
    <test-file-stub
      alt="Mail"
      loading="lazy"
    />
  </a>
  <a
    aria-label="Twitter"
    className="c1"
    href="https://twitter.com/elebouch"
    rel="noopener noreferrer"
    target="_blank"
  >
    <test-file-stub
      alt="Twitter"
      loading="lazy"
    />
  </a>
  <a
    aria-label="Sourcerer"
    className="c1"
    href="https://sourcerer.io/eleboucher"
    rel="noopener noreferrer"
    target="_blank"
  >
    <test-file-stub
      alt="Sourcerer"
      loading="lazy"
    />
  </a>
  <a
    aria-label="Sourcerer"
    className="c1"
    href="https://dribbble.com/eleboucher"
    rel="noopener noreferrer"
    target="_blank"
  >
    <test-file-stub
      alt="Dribbble"
      loading="lazy"
    />
  </a>
</div>
`;
